src/spyglass/common/common_behav.py

```diff
@@ -424,25 +424,22 @@ def get_abs_path(cls, key: Dict):
         nwb_video_file_abspath : str
             The absolute path for the given file name.
         """
-        video_dir = pathlib.Path(os.getenv("SPYGLASS_VIDEO_DIR", None))
-        assert video_dir is not None, "You must set SPYGLASS_VIDEO_DIR"
-        if not video_dir.exists():
-            raise OSError("SPYGLASS_VIDEO_DIR does not exist")
+        video_path_obj = pathlib.Path(video_dir)
         video_info = (cls & key).fetch1()
         nwb_path = Nwbfile.get_abs_path(key["nwb_file_name"])
         nwbf = get_nwb_file(nwb_path)
         nwb_video = nwbf.objects[video_info["video_file_object_id"]]
         video_filename = nwb_video.name
         # see if the file exists and is stored in the base analysis dir
         nwb_video_file_abspath = pathlib.Path(
-            f"{video_dir}/{pathlib.Path(video_filename)}"
```

Comment:
Would:
```python
video_path_obj / pathlib.Path(video_filename)
```
be cleaner?

Reply:
Sure - I was going for minimal editing, but I'll add this with my next commit